src/main/java/core/basesyntax/model/Apple.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.model;
+
+public class Apple extends Fruit {
+    private static final String NAME_OF_FRUIT = "apple";
+
+    public Apple(int quantity) {
+        super(NAME_OF_FRUIT, quantity);
```

Comment:
you dont need those classes. Just store a name of a fruit in a storage and its amount
potentially there can be a lot more fruits. with this approach you would create a class for each fruit